src/main/java/com/github/nylle/javafixture/SpecimenFactory.java

```diff
@@ -43,10 +51,38 @@ public SpecimenFactory(Context context) {
         if(type.isInterface()) {
             return new InterfaceSpecimen<>(type, context, this);
         }
+        if(isTimeSpecimen(type)) {
+            return new TimeSpecimen<>( type );
+        }
+        if(Temporal.class.isAssignableFrom(type)) {
+            return new TemporalSpecimen<>( type, context );
+        }
 
         return new ObjectSpecimen<>(type, context, this);
     }
 
+    private static boolean isTimeSpecimen( Class type ) {
```

Comment:
Maybe move that method to `ReflectionHelper`? Then it needs a different name though...